react/components/Menu/index.js

```diff
@@ -174,6 +174,7 @@ class Menu extends Component {
                     className={menuHeight ? 'overflow-auto' : ''}>
                     {options.map((option, index) => (
                       <button
+                        disabled={option.disabled}
                         key={index}
                         className="flex justify-between items-center t-body ph6 h-regular pointer hover-bg-muted-5 ma0 bg-transparent bn w-100 tl"
```

Comment:
The button still has the `pointer` class when disabled.
Is it necessary for a button at all? 🤔  My guess is that you could remove it just fine.

Reply:
Maybe hover classes should be removed on disabled too. What do you think?

Reply:
Agreed on removing `pointer`! But not sure about hover 🤔